spec/features/sdr_deposit_spec.rb

```diff
@@ -33,8 +33,8 @@
 
     # Tests existence of technical metadata
     expect(page).to have_content 'Technical metadata'
-    find('#document-techmd-head').click # expand the technical metadata accordion
-    file_listing = find_all('#document-techmd-section > ul > li')
+    find('#document-techmd-heading').click # expand the technical metadata accordion
```

Comment:
The improved accessability of https://github.com/sul-dlss/argo/commit/22df20679c1b15a868943f71d2df46b0cdaef450#diff-3441850a1c0bda539051183d4a19642a25f57714d17c3f8847aa52024b568dd3R3 should enable us to change this to `click_button 'Technical metadata'`